iphone/package.json

```diff
@@ -19,7 +19,7 @@
 	"minIosVersion": "10.0",
 	"minWatchosVersion": "2.0",
 	"vendorDependencies": {
-		"xcode": ">=9.0 <=12.x",
+		"xcode": ">=11.0 <=12.x",
 		"ios sdk": ">=10.0 <=14.x"
```

Comment:
I know we want to enforce Xcode 11 for xcframework support - but does that necessitate a bump in min iOS we can target? Or are we just bumping this now anyways because Apple requires iOS 13+ for any new apps or updates?

Reply:
Or is this due to some difference between this value and the `minIosVersion` value? I can never remember why they are different. cc @cb1kenobi

Reply:
Min iOS we can target is decided by "minIosVersion": "10.0". 
This is iOS SDK. If we use Xcode 11, which means used iOS SDK is 13.0. If we use Xcode 12, iOS SDK is 14.0. But this does not mean we can not set minimum target iOS 10. Though this entry seems redundant to me. Because if you have set Xcode 11, which means you are using iOS SDK 13.0.
I am not sure how it is getting used in Titanium SDK.